examples/autofill-utils/src/autofill-utils.rs

```diff
@@ -150,7 +150,7 @@ fn run_update_address(store: &Store, filename: String) -> Result<()> {
     let guid = address_fields.guid.clone();
 
     println!("Making `update_address` api call for guid {}", guid);
-    Store::update_address(store, &address_fields)?;
+    Store::update_address(store, address_fields)?;
```

Comment:
If you like, you should be able to make these take references by annotating the argument with `[ByRef]` in the UDL, ref [these docs](https://mozilla.github.io/uniffi-rs/udl/builtin_types.html#built-in-types) and [this example](https://github.com/mozilla/uniffi-rs/blob/4812fc2494d6db6e1800c0da8f7eba69b235451c/examples/sprites/src/sprites.udl#L3).

Reply:
Thanks! I did miss that in the docs, but I think this is OK for now - the `update` and `insert` functions are both consistent now, which really was the main goal. I don't think it makes any difference to the kotlin code, right?

Reply:
Right, shouldn't make any difference to consumers.